public/locales/en/translation.json

```diff
@@ -11,6 +11,23 @@
     "date_afternoon": "Good afternoon! It's {{date, DATE_HUGE}}",
     "date_evening": "Good evening! Today was the {{date, DATE_HUGE}}"
   },
+  "user": {
+    "email": {
+      "name": "email",
+      "placeholder": "Type your email",
+      "validation": {
+        "requied": "Email is required",
+        "format": "Invalid email format"
```

Comment:
To chyba będziemy się powtarzać, bo już zrobiłam teksty dla walidacji. Spójrz, proszę na najnowsze zmiany z maina. Podoba mi się, jak to zrobiłaś, chyba Twoje rozwiązanie jest lepsze :) Daj znać.

Reply:
Własnie jak wystawiłam Pr to jeszcze nie było tego, dzisiaj domergowałam zmiany i rzeczywiście się powtarzamy. Mi to bez różnicy, które rozwiązanie. Oby było mniej "szukania" póżniej. Także decyduj

Reply:
ok, może skorzystamy z Twojego rozwiązania? gdy zmergujesz to ja u siebie poprawię i wystawię pr :)

Reply:
ok spoko :-)